views/hoc/withRouter.jsx

```diff
@@ -0,0 +1,12 @@
+import { useLocation, useParams } from 'react-router-dom'
+
+export const withRouter = (Component) => {
```

Comment:
why do we need this? We can use the hooks directly in the components that need access to this data.